verilog/analysis/verilog_linter.cc

```diff
@@ -332,10 +333,51 @@ absl::Status PrintRuleInfo(std::ostream* os,
   return absl::OkStatus();
 }
 
+static void AppendCitation(CustomCitationMap& citations,
```

Comment:
Output parameters should [usually be at the end](https://google.github.io/styleguide/cppguide.html#Inputs_and_Outputs);

Also it is custom in this code-base to have output parameters pointers instead of references, but given that this is a static helper function used right afterwards, the confusion about it being a reference is probably not that big, so if you _really_ prefer the reference, keep it :)